schema.sql

```diff
@@ -11,3 +11,23 @@ CREATE TABLE animals (
 /* Day two the used queries */
 
 ALTER TABLE animals ADD COLUMN species VARCHAR(50);
+
+/* Day three the used QUERIES */
+
+-- add owners table
+
+CREATE TABLE owners ( id SERIAL PRIMARY KEY, full_name VARCHAR(255), age INTEGER )
+
+--add species table
+
+CREATE TABLE species ( id SERIAL PRIMARY KEY, name VARCHAR(255) )
+
+--modify animals table
+
+ALTER TABLE animals
+  DROP COLUMN species,
+  ADD COLUMN species_id INTEGER REFERENCES species(id)
+  ADD COLUMN owner_id INTEGER REFERENCES owners(id)
+
+  ALTER TABLE animals
+  ALTER COLUMN id SET DEFAULT nextval('animals_id_seq')
```

Comment:
- [ ] In SQL, you ought to use a `semi-colon(;)` to end a statement. `The semi-colon(;)` tells the programme when it is being run that it is the end of a statement. Kindly use the `semi-colon(;)` to end all statements in this file. 👍 😎

Reply:
Thanks!